.github/workflows/ci.yml

```diff
@@ -222,7 +222,8 @@ jobs:
           plutil -lint ./scripts/Info.plist
           mkdir -p SystemBridge.app/Contents/{MacOS,Resources}
           cp scripts/Info.plist SystemBridge.app/Contents
-          cp -R out/* SystemBridge.app/Contents/Resources
+          cp out/{system-bridge,system-bridge-tray,xdg-open,traybin/tray_darwin_release} SystemBridge.app/Contents/MacOS
+          cp -R out/system-bridge-circle.* SystemBridge.app/Contents/Resources
```

Comment:
So instead of going into the `Resources`, the executables should go into `MacOS`? Makes sense I guess.

Theres also `Frameworks` or `PlugIns` which might need the extra files/executables to go into.